### PR TITLE
refactor(api): rejig OID generation

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#366

### Dependencies

- Upstream: none
- Downstream: https://github.com/metriport/metriport-internal/pull/485

### Description

- OID generation with sequence per customer
- add account/customer init
  - ⚠️ this endpoint doesn't has auth, relies on internal's endpoint `/internal` created on API GW to close public access
- add API DNS entry on private zone
- update README w/ acc init & API Key setup

### Release Plan

- ⚠️ Contains DB migration
- ⚠️ Contains Infra update (auto applied upon merge)

### CODE REVIEW

File: utils/src/load-test.sh
Comment (leite08) [inline utils/src/load-test.sh:1]: Forgot to add this before. Used to generate a load on the API Server to create patients for a given Org. Went up to 100/s and it worked just fine.

Comment (leite08) [reply utils/src/load-test.sh:1]: @Goncharo @Orta21 FYI, maybe useful in other situations as well.

File: README.md [hunk 1]
```diff
@@ -434,7 +434,17 @@ Check the scripts on the folder's [package.json](https://github.com/metriport/me
...
+
+To do it manually:
+
+1. Login to the AWS console;
+1. Go to API Gateway;
+1. Create a Usage Plan if you don't already have one;
+1. Create an API Key;
+1. Add the newly created API Key to a Usage Plan.
+
+Now you can make requests to endpoints that require the an API Key by setting the `x-api-key` header.
```
[the remark this answers is not in the document]
Comment (leite08) [reply README.md]: I've updated it, @Goncharo - can you check it, please?

**Review by Goncharo — APPROVED** [review]
🚢 🚢 🚢

**Review by Orta21 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -434,7 +434,20 @@ Check the scripts on the folder's [package.json](https://github.com/metriport/me
...
+
+1. Login to the AWS console;
+1. Go to API Gateway;
+1. Create a Usage Plan if you don't already have one;
+1. Create an API Key;
+   - the `value` field must follow this pattern: base 64 of "`<KEY>:<UUID>`", where:
+   - `KEY` is a random key (e.g., generated with `nanoid`); and
+   - `UUID` is the customer ID (more about this on [Initialization](#initialization))
+1. Add the newly created API Key to a Usage Plan.
+
```
[the remark this answers is not in the document]
Comment (leite08) [reply README.md:449]: Markdown does numbering automatically when rendering, this way we don't need to keep them in order as we update the contents of the file 🎉 

image

[the remark this answers is not in the document]
Comment (leite08) [reply README.md:449]: good thinking and attention, though! 🕵️

**Pull request merged**